includes/Experiments.php

```diff
@@ -350,6 +350,7 @@ public function get_experiments() {
 				'label'       => __( 'Custom Meta Boxes', 'web-stories' ),
 				'description' => __( 'Enable support for custom meta boxes', 'web-stories' ),
 				'group'       => 'editor',
+				'default'     => true,
```

Comment:
Why not just remove this element from the array?

Reply:
This way we could easily hide the feature again in case there are any unexpected issues.

See https://github.com/google/web-stories-wp/blob/main/docs/feature-flags.md#enabling-feature-flags-by-default